utils: support additional config within different location

the build support that, so allow overriding configuration
location also in import/export.

Signed-off-by: Alon Bar-Lev <alonbl@redhat.com>

---

--- a/jasperreports-server-cp-5.5.0-bin/buildomatic/bin/js-import-export.sh	2014-01-16 01:41:57.058840500 +0200
+++ b/jasperreports-server-cp-5.5.0-bin/buildomatic/bin/js-import-export.sh	2014-01-16 01:42:13.853845937 +0200
@@ -34,7 +34,7 @@ fi
 
 # Additional config folder. This will be used to
 # get js.jdbc.properties from buildomatic setup
-export ADDITIONAL_CONFIG_DIR=$BASEDIR/build_conf/default
+export ADDITIONAL_CONFIG_DIR=${ADDITIONAL_CONFIG_DIR:-$BASEDIR/build_conf/default}
 
 
 # Locate the java binary bundled with installer
